great_expectations/render/renderer/column_section_renderer.py

```diff
@@ -109,6 +131,9 @@ def render(self, evrs, section_name=None, column_type=None):
                     content_blocks.append(getattr(self, content_block_function_name)(evrs))
             except Exception as e:
                 logger.error("Exception occurred during data docs rendering: ", e, exc_info=True)
+                exception_traceback = traceback.format_exc()
+                exception_message = f'{type(e).__name__}: "{str(e)}".  Traceback: "{exception_traceback}".'
+                raise GreatExpectationsError(exception_message)
```

Comment:
I don't remember the full context on why we suppress errors here, but I do remember it being important. This re-raise makes me nervous.

Reply:
Let's discuss -- together with @roblim -- here is the background: My understanding was that @roblim did not want to reraise in order to give the users of init a smooth experience.  I found it to be more harmful when the configuration is wrong, content is missing, and there is no information.  From my experiments, all these errors should not have been recoverable, and I felt it would be better to stop the experience and alert the user to fix the configuration.  Now, whatever is wrong with the configuration is explained in as specific a detail as possible, making fixes quick, and further experience more expected overall.  Happy to discuss.  thank you. /cc @Aylr @roblim @jcampbell